src/pages/using-spark/foundations/design-tokens.mdx

```diff
@@ -3,21 +3,62 @@
 ---
 import SassVarTable from '../../../components/SassVarTable';
 
-# Design Tokens
+<h1 class="docs-b-h1 sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
+  Design Tokens
+</h1>
 
-Spark provides design tokens that store our design decisions.
-These tokens can be retrieved in different formats for different devices.
-These variables provide global settings for components in the system.
-For web, almost every aspect of each Spark component is
-configurable with a Sass variable.
+Our Design Tokens are a catalog from which we base all our visual and branding choices.
 
-** Override the Sass settings by: **
-1. Creating a new Sass file.
-2. Importing it before Spark's main Sass file.
-3. Assigning a new value to the variable you want to override.
+It's a central database of our core styles like colors, typography, spacing, sizing, and much more.
+
+Spark Design Tokens currently generate style files for web, iOS, and Android platforms.
+
+## Benefits
+Design Tokens allow more projects to incorporate our design decisions.
+
+<h3 class="docs-b-h5 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure">
+  Designer Perks
+</h3>
+
+- Encourages visual consistency and design consensus across products.
+- Creates easier syncing with brand guidelines.
+- Encourages better support for multi-platform projects.
+- Promotes clear communication with developers and other designers.
+
+<h3 class="docs-b-h5 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure">
+  Developer Perks
+</h3>
+
+- Maintains sync with current style updates while minimizing impact to code.
+- Enables Android and iOS projects to access style variables from Spark.
+- Supports future-friendly development. Design Tokens can generate style files for any possible platform in the future.
+
+## How to Use Our Design Tokens
+
+<h3 class="docs-b-h5 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure">
+ For Designers
+</h3>
+
+Use token names to communicate the specific style values of your design.
+
+Token names depict style choices that portray color, spacing, typography, and more.
+
+For example, the token `sprk-purple-deep` represents a specific shade of purple. Style consensus makes it clear to everyone which purple you mean.
+
+<h3 class="docs-b-h5 sprk-b-TypeDisplayFive sprk-u-mtl sprk-u-mbm sprk-u-Measure">
+ For Developers
+</h3>
+
+For HTML, React, and Angular projects, follow our [install guides](/installing-spark). They show you how to import the correct Sass file.
+
+** How to Change Spark's Default Style Settings: **
+1. Create a new Sass file.
```

Comment:
The `<ul>` up above has `sprk-b-List sprk-b-List--indented` applied, but not the `<ol>` here. Not sure how we are applying that to the markdown bulleted list, but we need to add the same thing to ordered lists.

Reply:
@yodasw16  this is how it is throughout the documentation. Would you expect this to change everywhere?

Reply:
gonna add it in, and then check how other pages are affected

Reply:
The numbers were very easy to parse in  the installation guides, but with this as default it looks awkward.

**What I'll do is add the indented styles specifically for the Design Token page.**

WITH CHANGES:
![image](https://user-images.githubusercontent.com/4342363/99411383-89c32980-28c1-11eb-936b-9b37672d49b6.png)
![image](https://user-images.githubusercontent.com/4342363/99411415-921b6480-28c1-11eb-9a78-cb542ccfd036.png)

--------------
BEFORE:
![image](https://user-images.githubusercontent.com/4342363/99411818-02c28100-28c2-11eb-8f3e-2795e04bbefb.png)